lxd/vm_qemu.go

```diff
@@ -990,6 +1040,10 @@ driver = "virtio-scsi-pci"
 bus = "qemu_pcie1"
 addr = "0x0"
 
+# Graphics card
```

Comment:
@stgraber was this required for the qmp work or just part of planned work to add GPU support?

Reply:
No, it's just another tweak to our machine type I had around to allow for people to play with GUI installs. Combined with `raw.qemu` all they need is to pass `-vnc :1` to have a working graphical console.